src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,11 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        String stringToPrint = "Name = " + this.value;
```

Comment:
Does this class represent a **name** or an **address**?
On that note, you might want to consider refactoring out these "magic" words.